### PR TITLE
new readme

### PR DESCRIPTION
**Pull request by Jordandiazfr**
Readme Updated  #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,55 @@
...
+- _TypeScript_
+- _@types/node_
+- _mocha_ (For testing)
+- _Chai_ (Test unitaires)
+- _ffmpeg_ (audio files)
+- _ytdl-core_
+
+### List of commands 📖
+
+| Version of Tob | Command avaliable       | Action                                                                   |
```
Comment (LuffyStar93) [inline README.md]: available*

File: README.md [hunk 2]
```diff
@@ -1,27 +1,55 @@
...
+| -------------- | ----------------------- | ------------------------------------------------------------------------ |
+| 0.1            | !ping                   | Returns pong!                                                            |
+| 0.2            | !lire , read            | Return a random article to read from hackernews                          |
+| 0.2            | !help                   | Return the list of commands                                              |
+| 0.3            | !roulette               | Starts a russian roulette game of six turns max                          |
+| 0.4            | !kaamelott              | Returns a message with a legendary quote from the french series kaamelot |
+| x.x            | !!play + 'youtube link' | Play a song from a youtube link in a voice channel                       |
+| x.x            | !stop                   | Stops playing the music tha tit started to play                          |
+| x.x            | !8ball                  | The magic ball that return yes and no to a question                      |
```
Comment (LuffyStar93) [inline README.md]: !8ball  + 'question'

File: README.md [hunk 3]
```diff
@@ -1,27 +1,55 @@
-# Discord Bot: Tob! 
+# Discord Bot ("Tob 🤖") v0.5
 
-![](https://i.ibb.co/3FtJNcG/tob.png)
+**Tob** is a bot from discord created to enhance your discord experience and to test a new programming architecture in typescript, node into a virtual environment ran by docker. 🐳
 
-__Created by Bart, Max et Jojo, Tob is gonna help you to know when is the next Train or Bus that you have to take to Go Home
-from one place to another (only valid in Paris and peripheria )__
+### Requeriments 📋
```
Comment (bjanik) [inline README.md:5]: *Requirements

File: README.md (hunk 1 above)
Comment (Jordandiazfr) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (Jordandiazfr) [reply README.md]: ok

**Review by LuffyStar93 — APPROVED** [review]
Good !

**Pull request merged**